go/engine/signup.go

```diff
@@ -122,7 +122,7 @@ func (s *SignupEngine) Run(m libkb.MetaContext) (err error) {
 		return err
 	}
 
-	if err = s.registerDevice(m, s.arg.DeviceName); err != nil {
+	if err = s.registerDevice(m, s.arg.DeviceName, s.arg.GenerateRandomPassphrase); err != nil {
```

Comment:
What's this random passphrase stuff in signup?

Reply:
"random passphrase" is the same as "no passphrase". I needed to thread it through to the point where we store it so we can decide to not use the keyring in this case.

Reply:
But isn't "no passphrase" a separate project?